data.sql

```diff
@@ -1,5 +1,12 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES ('Agumon', '2020-02-03', 0, TRUE, 10.23, 'Unknown'),
+       ('Gabumon', '2018-11-15', 2, TRUE, 8, 'Unknown'),
+       ('Pikachu', '2021-01-07', 1, FALSE, 15.04, 'Unknown'),
+       ('Devimon', '2017-05-12', 5, TRUE, 11, 'Unknown'),
```

Comment:
- [ ] @DevKaranJ Kindly check the data given,you have entered incorrect figures in your database, below is a screenshot of the data given.
![image](https://github.com/DevKaranJ/vet-clinic/assets/61408860/0a60cf99-0f3f-4200-9618-2dcb18ab6690)